pkg/localstore/localstore_test.go

```diff
@@ -567,6 +567,6 @@ func TestDBDebugIndexes(t *testing.T) {
 	}
 
 	// assert that there's a pin and gc exclude entry now
-	testIndexCounts(t, 1, 1, 1, 1, 1, 1, 1, indexCounts)
+	testIndexCounts(t, 1, 1, 0, 1, 1, 1, 1, indexCounts)
```

Comment:
why has this been changed?

Reply:
Previousl when you setAccesss, we add the chunk to gcIndex even of it is pinned. Now for a pinned chunk, even after a setAccess, the gcIndex should not be incremented as we have added our check their.